readme.md

```diff
@@ -50,7 +50,7 @@
 - [Border Gradients](https://github.com/cossssmin/tailwindcss-border-gradients) - Adds `border-image` gradient utilities.
 - [RFS](https://github.com/aerni/tailwindcss-rfs) - Adds [`RFS`](https://github.com/twbs/rfs) utilities.
 - [Font Variant Numeric](https://github.com/philippbosch/tailwindcss-font-variant-numeric) - Adds `font-variant-numeric` utilities.
-- [Dark Mode](https://github.com/ChanceArthur/tailwindcss-dark-mode) - Adds `dark` variants based on the `prefers-color-scheme` media query.
+- [Dark Mode](https://github.com/danestves/tailwindcss-darkmode) - Adds `dark` variants based on the class of the `body` by default `.dark-mode`
```

Comment:
Can you only describe the plugin and not its behavior? In this case, a preferable description would be
> Adds `dark` variants based on CSS classes.

Reply:
Done! Thanks